app/controllers/concerns/curation_concerns/single_use_links_controller_behavior.rb

```diff
@@ -44,7 +44,7 @@ def authorize_user!
       end
 
       def asset
-        @asset ||= ActiveFedora::Base.load_instance_from_solr(params[:id])
+        @asset ||= ActiveFedora::Base.find(params[:id])
```

Comment:
No, I didn't, but it would make show pages for FileSets reliant on Fedora and did fix that last month.